forge/test/operators/utils/utils.py

```diff
@@ -90,6 +91,15 @@ def set_math_fidelity(math_fidelity: MathFidelity):
         compiler_cfg.default_math_fidelity = math_fidelity
 
 
+class DeviceUtils:
+    '''Utility functions for Forge verification'''
+
+    @staticmethod
+    def warm_reset():
+        reset_command = "/home/software/syseng/wh/tt-smi -lr all wait -er"
```

Comment:
I recommend to make it a bit more general, because around the corner is new chip type which will potentially have new warm reset command :))

Reply:
If we need to distinguish devices we would have to resolve also #342